components/qt5/Makefile

```diff
@@ -0,0 +1,99 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2014 EveryCity Ltd. All rights reserved.
+# Copyright 2015 Alexander Pyhalov
+#
+
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           qt
+COMPONENT_VERSION=        5.8.0
+COMPONENT_VERSION_MJR=    5.8
+COMPONENT_FMRI=           library/qt5
+COMPONENT_PROJECT_URL=    https://www.qt.io/
+COMPONENT_SUMMARY=        Qt cross-platform application and UI framework
+COMPONENT_CLASSIFICATION= Desktop (GNOME)/Libraries
+COMPONENT_SRC=		$(COMPONENT_NAME)-everywhere-opensource-src-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+#COMPONENT_ARCHIVE_HASH=	\
```

Comment:
userland-fetch  throws a memory error:
File "/usr/lib/python2.7/gzip.py", line 338, in _add_read_data
    self.extrabuf = self.extrabuf[offset:] + data
MemoryError